route53_crossfeed_app.tf

```diff
@@ -0,0 +1,301 @@
+# ------------------------------------------------------------------------------
+# Resource records that support Crossfeed site cloudfront endpoints and application.
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# Prod entries
+# ------------------------------------------------------------------------------
+
+resource "aws_route53_record" "crossfeed_prod_A" {
+  provider = aws.route53resourcechange
+
+  alias {
+    name                   = "d2urclllpa4h1x.cloudfront.net."
+    zone_id                = aws_route53_zone.cyber_dhs_gov.zone_id
+    evaluate_target_health = false
```

Comment:
I noticed that @felddy used a value of `false` [here](https://github.com/cisagov/cool-dns-cyber.dhs.gov/blob/develop/route53_federalist_app.tf#L11).  I'm ignorant of the difference, but can @hillaryj or @felddy explain why it should be different in these two cases?